README.md

```diff
@@ -207,31 +162,136 @@ Users want to determine several types of the formatting width as following
+  * `years: "long" | "short" | "narrow"`: The style to be used for formatting years.
+  * `yearsDisplay: "always" | "auto"`: Whether to always display years, or only if nonzero.
+  * `months: "long" | "short" | "narrow"`: The style to be used for formatting months.
+  * `monthsDisplay: "always" | "auto"`: Whether to always display months, or only if nonzero.
+  * `weeks: "long" | "short" | "narrow"`: The style to be used for formatting weeks.
+  * `weeksDisplay: "always" | "auto"`: Whether to always display weeks, or only if nonzero.
+  * `days: "long" | "short" | "narrow"`: The style to be used for formatting days.
+  * `daysDisplay: "always" | "auto"`: Whether to always display days, or only if nonzero.
+  * `hours: "long" | "short" | "narrow" | "numeric" | "2-digit"`: The style to be used for formatting hours.
+  * `hoursDisplay: "always" | "auto"`: Whether to always display hours, or only if nonzero.
+  * `minutes: "long" | "short" | "narrow" | "numeric" | "2-digit"`: The style to be used for formatting minutes.
+  * `minutesDisplay: "always" | "auto"`: Whether to always display minutes, or only if nonzero.
+  * `seconds: "long" | "short" | "narrow" | "numeric" | "2-digit"`: The style to be used for formatting seconds.
+  * `secondsDisplay: "always" | "auto"`: Whether to always display seconds, or only if nonzero.
+  * `milliseconds: "long" | "short" | "narrow" | "numeric"`: The style to be used for formatting milliseconds.
+  * `millisecondsDisplay: "always" | "auto"`: Whether to always display milliseconds, or only if nonzero.
+  * `microseconds: "long" | "short" | "narrow" | "numeric"`: The style to be used for formatting microseconds.
+  * `microsecondsDisplay: "always" | "auto"`: Whether to always display microseconds, or only if nonzero.
+  * `nanoseconds: "long" | "short" | "narrow" | "numeric"`: The style to be used for formatting nanoseconds.
+  * `nanosecondsDisplay: "always" | "auto"`: Whether to always display nanoseconds, or only if nonzero.
+  * `fractionalDigits: number`: The number of fractional digits to be included in the output, applies automatically to the entire set of consecutive time units with style `"numeric"` at the end.
+
+##### Default values
+
+* The per-unit style options default to the value of `style` for all styles except `"digital"`, for which units years till days default to `"narrow"` and hours till nanoseconds default to `"numeric"`.
+* The per-unit display options default to `"auto"` if the corresponding style option is `undefined` and `"always"` otherwise.
+
+#### Notes
+
+* If for any given locale and unit, the representation of the `"narrow"` style is completely unique, then the same repsresentation may be used for `"short"`.
+* Any unit with the style `"numeric"` followed by a unit of style `"numeric"` or `"2-digit"` should act like the style `"2-digit"` was used instead.
```

Comment:
I believe that the statement above is backwards. 
```suggestion
* Any unit with the style `"numeric"` preceded by a unit of style `"numeric"` or `"2-digit"` should act like the style `"2-digit"` was used instead.  For example, `{hours: 'numeric', minutes: 'numeric'}` can produce output like "3:08".
```

Reply:
What happens with weird styles, e.g. `{hours: numeric, minutes: 'long'}`. Error? Does "long" get upgraded to "2-digit"? Something else?  (I repeated this comment in the spec text too)

Reply:
you're right, I will merge the commit to make the explainer more accurate. In the meantime, could you make an issue for the "weird style" problem? That sounds like something we should look into.

Reply:
Done. #66.